### ISSUE TITLE
configure support for cuda on arm64

### ISSUE THREAD

**Issue by frpunzalan** [kaldi-asr/kaldi#2577]
Hi,

I was trying to compile kaldi on a 64-bit Tegra device and ran into some errors:

/usr/local/bin/ld: cannot find -lcublas
...

I looked into the generated kaldi.mk and it seems that compiler couldn't find the correct library path. The linker was pointing to /usr/local/cuda/lib instead of /user/local/cuda/lib64. Looking at the configure script, it seems that the makefiles/cuda_32bit.mk was included to kaldi.mk instead of the 64-bit version.

Fixing the kaldi.mk to let the linker point to lib64 fixes the compile error. For future releases, maybe we can add support for cuda on arm64. Here is the configure script change that will probably generate the correct kaldi.mk for cuda on arm64:

```diff
diff --git a/src/configure b/src/configure
index 2f506b9..a954583 100755
--- a/src/configure
+++ b/src/configure
@@ -454,6 +454,8 @@ function configure_cuda {
       else
         cat makefiles/cuda_64bit.mk >> kaldi.mk
       fi
+    elif [ "`uname -m`" == "aarch64" ]; then
+      cat makefiles/cuda_64bit.mk >> kaldi.mk
     elif [ "`uname -m`" == "ppc64le" ]; then
       cat makefiles/cuda_64bit.mk >> kaldi.mk
     else
```

**Comment by danpovey** [comment]
Thanks.  Any chance you can make a PR for that change

**Comment by frpunzalan** [comment]
Sure. Will do once I tested it out.

**Comment by frpunzalan** [comment]
tested the configure edits on an arm64 device and it compiled. also ran make test on src/cudamatrix and checked the log to make sure that gpu was used.

I submitted a pr just now.

Cheers,
Rusty

**Comment by kkm000** [comment]
Fixed in #2578
